platform_umbrella/rel/overlays/bin/make_admin

```diff
@@ -11,4 +11,5 @@ if [[ -z "${EMAIL}" ]]; then
     exit 1
 fi
 
-"${DIR}/home_base" eval "HomeBase.Release.make_admin\(\"${EMAIL}\"\)"
+MAKE_ADMIN_CMD=$(printf 'HomeBase.Release.make_admin!("%s")' "${EMAIL}")
```

Comment:
It was missing the `!` at the very least. This is also congruent with how we eval these commands in the other init script e.g. https://github.com/batteries-included/batteries-included/blob/master/platform_umbrella/rel/overlays/bin/start#L12.